src/config/cloudinaryconfig.ts

```diff
@@ -0,0 +1,11 @@
+import { v2 as cloudinary } from "cloudinary";
```

Comment:
File name convention should be kebab-case. So the file name should b cloudinary-config.ts